### PR TITLE
[10.x.x] Added XR rendering checkbox to URP camera's inspector.

### PR DESCRIPTION
**Pull request by thomas-zeng**
### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [x] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [ ] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR.
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Address https://fogbugz.unity3d.com/f/cases/1261612/
This PR added XR rendering checkbox to URP camera inspector.
image

---
### Testing status

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally
- [x] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
- None

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs [hunk 1]
```diff
@@ -108,7 +108,9 @@ public Matrix4x4 GetGPUProjectionMatrix(int viewIndex = 0)
         public bool isHdrEnabled;
         public bool requiresDepthTexture;
         public bool requiresOpaqueTexture;
-
+#if ENABLE_VR && ENABLE_XR_MODULE
+        public bool xrRendering;
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs:112]: we recently deprecated `isStereoEnabled`, shouldn't we use that instead?

Comment (thomas-zeng) [reply com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs:112]: This is done by purpose. We want to deprecate isStereoEnabled and switch users to use xr.enabled instead. xr.enabled is populated by XR system every frame. 
We believe deprecation can make it clear to users that they should use xr.enabled for runtime XR check. As for xrRendering, this is populated by camera inspector UI, not runtime XRSystem.

**Comment by thomas-zeng** [comment]
To QA,
We guarded this UI within XR define so that non-XR won't show the UI.
Please verify without the com.unity.modules.xr, this target eye UI is not shown.
Also I think we should verify this checkbox value persist when user add/remove com.unity.modules.xr, not resetting.

**Comment by phi-lira** [comment]
@thomas-zeng URP stereo test is failing. Can you verify if this is real issue?

**Comment by thomas-zeng** [comment]
@phi-lira Fixed the failed stereo test. The issue is that the test's camera is not set for URP and is missing URPAdditionalCameraData component. Added that component to the camera asset.
This PR now only fails HDRP tests which I believe is caused by known issues in DXR.

**Comment by thomas-zeng** [comment]
@phi-lira I believe this PR is ready to land. All ABV failures are related to HDRP DXR.

**Pull request merged**